app/AdminUser.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class AdminUser extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $table = 'admin_users';
+
+    protected $fillable = ['username', 'password', 'fullname', 'created_at', 'updated_at'];
+
+    protected $hidden = [
+        'password'
+    ];
+
+    /**
+     * Get all category for AdminUser .
+     *
+     * @return array
```

Comment:
not return array, please check the Laravel core to get the return class. And please fix for all. Thanks